clair/values.yaml

```diff
@@ -0,0 +1,79 @@
+# Default values for clair.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+logLevel: info
+insecureTls: false
+image:
+  repository: quay.io/coreos/clair-git
+  tag: latest
```

Comment:
Use the last available LTS and not latest please.
If you use latest the chart versioning become unreliable as you cannot be sure about the matching of the config and the application.